components/form/FormItem.tsx

```diff
@@ -159,15 +159,10 @@ function FormItem(props: FormItemProps): React.ReactElement {
 
     // ======================== Status ========================
     let mergedValidateStatus: ValidateStatus = '';
-    if (validateStatus !== undefined) {
-      mergedValidateStatus = validateStatus;
-    } else if (meta && meta.validating) {
-      mergedValidateStatus = 'validating';
-    } else if (!help && mergedErrors.length) {
-      mergedValidateStatus = 'error';
-    } else if (meta && meta.touched) {
-      mergedValidateStatus = 'success';
-    }
+    if (validateStatus !== undefined) mergedValidateStatus = validateStatus;
+    else if (meta?.validating) mergedValidateStatus = 'validating';
+    else if (meta?.errors?.length) mergedValidateStatus = 'error';
```

Comment:
mergedErrors ？

Reply:
Form.Item with `help` 
`mergedErrors = toArray(help);`

Reply:
should keep `{ }`.

Reply:
Done